job-service/src/main/java/com/hpe/caf/services/job/api/generated/JobsApiServiceImpl.java

```diff
@@ -89,7 +89,7 @@ public Response cancelJobs(final String partitionId, final String jobIdStartsWit
     {
 
         final int successfulCancellations = JobsCancel.cancelJobs(partitionId, jobIdStartsWith, label, filter);
-        return Response.ok(String.format("Successfully cancelled %s jobs", successfulCancellations)).build();
+        return Response.ok().entity(String.format("Successfully cancelled %s jobs", successfulCancellations)).build();
```

Comment:
```suggestion
        return Response.ok().entity(String.format("Successfully canceled %s jobs", successfulCancellations)).build();
```
This should be US spelling unfortunately